src/VisualStudio/IntegrationTest/TestUtilities/InProcess/SolutionExplorer_InProc.cs

```diff
@@ -414,6 +414,37 @@ public void AddFile(string projectName, string fileName, string contents = null,
             }
         }
 
+        /// <summary>
+        /// Adds a new standalone file to the Miscellaneous Files workspace.
+        /// </summary>
+        /// <param name="fileName">The name of the file to add.</param>
+        public void AddStandaloneFile(string fileName)
+        {
+            string itemTemplate;
+
+            var extension = Path.GetExtension(fileName).TrimStart('.').ToLowerInvariant();
```

Comment:
:bulb: Might as well avoid the `.TrimStart('.')` and put the dots in the switch cases.